This patch removes mp3 decoder cap from auddec elements. mp3 decoder is not part of tspa combo. If you are adding mp3 decoder support in your combo then do not apply this patch.

diff -uNr ticodecplugin/src/gsttiauddec1.c ticodecplugin.new/src/gsttiauddec1.c
--- ticodecplugin/src/gsttiauddec1.c	2009-09-05 00:04:51.000000000 -0500
+++ ticodecplugin.new/src/gsttiauddec1.c	2009-09-15 03:28:28.000000000 -0500
@@ -85,7 +85,7 @@
     GST_PAD_ALWAYS,
     GST_STATIC_CAPS
     ("audio/mpeg, "
-        "mpegversion = (int) { 1, 4 }")
+        "mpegversion = (int) { 4 }")
 );
 
 /* Define source (output) pad capabilities.  Currently, RAW is supported. */
diff -uNr ticodecplugin/src/gsttiauddec.c ticodecplugin.new/src/gsttiauddec.c
--- ticodecplugin/src/gsttiauddec.c	2009-09-05 00:04:51.000000000 -0500
+++ ticodecplugin.new/src/gsttiauddec.c	2009-09-16 07:51:24.000000000 -0500
@@ -85,7 +85,7 @@
     GST_PAD_ALWAYS,
     GST_STATIC_CAPS
     ("audio/mpeg, "
-        "mpegversion = (int) { 1, 4 }")
+        "mpegversion = (int) { 4 }")
 );
 
 /* Constants */
